NEWS.md

```diff
@@ -6,6 +6,8 @@
 * Added `read_target_file()` function for reading in individual target data files.
 * Added target data validation checks:
   - `check_target_file_name()`: that a hive-partitioned target data file name can be correctly parsed.
+  - `check_target_dataset_unique()`: that a single unique target dataset exists for a given target type.
+  - `check_target_dataset_file_ext_unique()`: that file(s) in a target dataset share a single unique file extension.
```

Comment:
I will suggest to  add the information that only recognize target data file(s) are tested.
For example, 
```suggestion
  - `check_target_dataset_file_ext_unique()`: that target file(s) (`time-series`, `oracle-output`) in a target dataset share a single unique file extension.
```

Reply:
I've added a comment in the NEWS. In the function documentation, the two target types are listed in the `target_type` argument options.